channels-rke2.yaml

```diff
@@ -852,4 +878,27 @@ releases:
       rke2-calico-crd:
         repo: rancher-rke2-charts
         version: v3.24.102
-    featureVersions: *featureVersions-v1
\ No newline at end of file
+    featureVersions: *featureVersions-v1
+  - version: v1.24.8-rc1+rke2r1
+    minChannelServerVersion: v2.6.7-alpha1
+    maxChannelServerVersion: v2.7.99
+    serverArgs: *serverArgs-v1-24-2-rke2r1
+    agentArgs: *agentArgs-v1-24-2-rke2r1
+    charts: &charts-v1-24-8-rc1-rke2r1
+      <<: *charts-v1
```

Comment:
I don't think charts-v1 is the correct anchor here. I think it should be `charts-v1-24-7-rke2r1`.